src/main/scala/cc/factorie/app/nlp/lexicon/AhoCorasick.scala

```diff
@@ -1,15 +1,5 @@
-/* Copyright 2014.
-   This file is part of "FACTORIE" (Factor graphs, Imperative, Extensible)
-   http://factorie.cs.umass.edu, http://github.com/factorie
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
+/*
+ * Copyright C 2014, 2015, Oracle and/or its affiliates. All rights reserved. 
```

Comment:
No. Our lawyers say this has to stay. We've file the contributors agreement which gives UMass & Andrew a license to our contributions but they are still our copyright.